specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/DatabaseSecurityAlertPolicies.json

```diff
@@ -0,0 +1,406 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-02-02-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```

Comment:
creating a securityAlertPOlicy is possible via the PUT method, but there is no DELETE. Its typical to provide the delete option for something which is created by the user.

Reply:
after further discussion, it seems like this resource type is a singleton. the customer can either turn on this policy for a database or turn it off. Ideally, it should be modeled as a property on the database itself which can be updated via Patch.

Reply:
it seems like in previous api-versions, this already existed as a separate child. and so RP would like to keep it consistent with that. And also, the PUT is not really creating the resource. The resource exists in a disabled state even when the PUT hasnt happened. So, in addition to PUT, service team will also add a PATCH.